opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -1938,7 +1938,13 @@ export const updateAttributeMetaResolved = async (context, user, initial, inputs
         if (currentValue?.id !== targetCreated?.internal_id) {
           // Delete the current relation
           if (currentValue?.standard_id) {
-            const currentRels = await listAllRelations(context, user, relType, { fromId: initial.id });
+            const currentRels = (await listAllRelations(context, user, relType, { fromId: initial.id }))
+              .map((rel) => ({
+                ...rel,
+                // we resolve from and to without need of an extra query
+                to: targetCreated,
+                from: initial,
+              }));
```

Comment:
this resolution is necessary in case we want to call `createContainerSharingTask` for the removed objects.

this is not the case for now (see my comment addec line 2062) but up to debate.
We chose to keep this resolution as it's harmless (no extra query).